README.md

```diff
@@ -36,4 +56,17 @@ url.search = "?q=canada&safe=off"
 print(url) # https://google.com/search?q=canada&safe=off
 ```
 
+## Performance
+
+We find that `can_ada` maybe be faster than `urllib`:
```

Comment:
`We find that `can_ada` may be faster than `urllib`:`

Reply:
Done.